packages/golden-layout/package.json

```diff
@@ -50,8 +50,8 @@
     "access": "public"
   },
   "devDependencies": {
+    "babelify": "^10.0.0",
```

Comment:
golden-layout should have `nanoid` in its normal dependencies. Looks like `shortid` wasn't added there initially which was a mistake